src/internal/deobfuscateAppleGPU.ts

```diff
@@ -82,7 +82,7 @@ export const deobfuscateAppleGPU = (
       gl.vertexAttribPointer(aPosition, 3, GL_FLOAT, false, 0, 0);
       gl.enableVertexAttribArray(aPosition);
 
-      gl.clearColor(1.0, 1.0, 1.0, 1.0);
+      gl.clearColor(1, 1, 1, 1);
```

Comment:
gl.clearColor has `GLclampf` parameters but seeing as all JavaScript numbers are floats anyways this should be fine.

https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext/clearColor

Reply:
Good to remember.